ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bfd.j2

```diff
@@ -8,4 +8,27 @@ router bfd
 {%     if router_bfd.multihop.interval is arista.avd.defined and router_bfd.multihop.min_rx is arista.avd.defined and router_bfd.multihop.multiplier is arista.avd.defined  %}
    multihop interval {{ router_bfd.multihop.interval }} min-rx {{ router_bfd.multihop.min_rx }} multiplier {{ router_bfd.multihop.multiplier }}
 {%     endif %}
+{%     if router_bfd.sbfd is arista.avd.defined %}
+   !
+   sbfd
+{%         if router_bfd.sbfd.local_interface is arista.avd.defined and router_bfd.sbfd.local_interface.protocols is arista.avd.defined %}
```

Comment:
I would do a check here also to see if either ipv4 or ipv6 is true so we don't render the command if none of them are true... something like:

```suggestion
{%         if router_bfd.sbfd.local_interface is arista.avd.defined and (router_bfd.sbfd.local_interface.protocols.ipv4 is arista.avd.defined(true) or router_bfd.sbfd.local_interface.protocols.ipv6 is arista.avd.defined(true)) %}
```